manager/src/lib.rs

```diff
@@ -532,31 +540,41 @@ impl CronManager {
     #[payable]
     pub fn register_agent(
         &mut self,
+        agent_account_id: Option<ValidAccountId>,
         payable_account_id: Option<ValidAccountId>
     ) {
         assert_eq!(self.paused, false, "Register agent paused");
 
-        // TODO: assert that attached deposit is enough to cover storage cost. This is to protect from storage attack.
```

Comment:
@mikedotexe maybe leave an updated comment?

Reply:
I removed this comment because we added the functionality in there, and have this below here where it has:
`assert!(deposit >= required_deposit, "Insufficient deposit. Please deposit {} yoctoⓃ to register an agent.", required_deposit.clone());`

So in my mind this was removing a TODO that had been moved to TO DONE 😏